docs/pages/versions/unversioned/sdk/notifications.mdx

```diff
@@ -393,7 +393,8 @@ This method makes requests to the Expo's servers. It can get rejected in cases w
 
 This function accepts an optional object allowing you to pass in configuration with the following fields:
 
-- **experienceId (_string_)** -- **passing is mandatory**. The ID of the experience to which the token should be attributed. Defaults to [`Constants.manifest.id`](/versions/latest/sdk/constants/#constantsmanifest) exposed by `expo-constants`. When building with EAS Build, or in the bare workflow, **this is required** and you must provide a value which takes the shape `@username/projectSlug`, where `username` is the Expo account that the project is associated with, and `projectSlug` is your [`slug` from **app.json**](../config/app.mdx#slug).
+- **projectId (_string_)** -- The ID of the project to which the token should be attributed. Defaults to [`Constants.expoConfig.extra.eas.projectId`](/versions/latest/sdk/constants/#easconfig) exposed by `expo-constants`. In case you are not using EAS Build, it will fallback to [`Constants.manifest.projectId`](/versions/latest/sdk/constants/#constantsmanifest).
```

Comment:
```suggestion
- **projectId (_string_)** -- The ID of the project to which the token should be attributed. Defaults to [`Constants.expoConfig.extra.eas.projectId`](/versions/latest/sdk/constants/#easconfig) exposed by `expo-constants`. If you are not using EAS Build, it will fallback to [`Constants.manifest.projectId`](/versions/latest/sdk/constants/#constantsmanifest).
```

One question regarding the "token should be attributed". Isn't it something automatically generated when EAS Build is configured inside a project in the Expo config file?

It's fine the way it reads right now but I thought if this value is automatically generated, we can be a little explicit about it or change the wording around "token should be attributed".

Reply:
So, the token that we're talking about here is the `ExpoPushToken`, which is different for every user and it's generated at runtime, `projectId` on the other hand is indeed generated when EAS Build is configured

Reply:
ah got it! My bad, I misunderstood. In that case, it's fine as it is.